lms/static/sass/application-extend1-rtl.scss

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
@Nickersoft, it looks like these files were removed from the repo but are still present on your machine. It should be safe to untrack and delete them locally.